### PR TITLE
Texts a la fredrik

### PR DESCRIPTION
**Pull request by Delodax**
By referring to the document in Trello and live discussion with Fredrik, this was the result!

### CODE REVIEW

File: client/react-hooks.js [hunk 1]
```diff
@@ -1,43 +1,43 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect } from 'react'
```
Comment (kaj) [inline client/react-hooks.js]: None of the changes in this file has anything with "Texts a la Fredrik" ...

File: server/export-to-ladok.js [hunk 2]
```diff
@@ -3,73 +3,73 @@ const {
   getLadokModules,
   sendGradesToLadok,
   getGrades
-} = require("../lib");
-const log = require("skog");
+} = require('../lib')
```
Comment (kaj) [inline server/export-to-ladok.js]: None of the changes in this file has anything to do with "Texts a la Fredrik".

File: client/react-hooks.js (hunk 1 above)
Comment (Delodax) [reply client/react-hooks.js]: @kaj : Hmm, was that part of the merge @exacs did?

File: server/export-to-ladok.js (hunk 2 above)
Comment (Delodax) [reply server/export-to-ladok.js]: @kaj : Hmm, was that part of the merge @exacs did?

Comment (exacs) [reply server/export-to-ladok.js]: Changes on this file are only reformat. We had some formatting inconsistencies there so it's been a bit messy to see the *actual* changes in the code. Should be better now… :)

**Pull request merged**